### DISCUSSION TITLE
Notifications on screen

### DISCUSSION THREAD

**Discussion by codeBaker1989** [Blackymas/NSPanel_HA_Blueprint#2177] [General]
Hi, is it possible to trigger a notification on the screen when a bin needs to be set out on the street? thanks.

**Comment by edwardtfn** [comment]
You can trigger notifications from HA: https://github.com/Blackymas/NSPanel_HA_Blueprint/blob/main/docs/howto.md#notification-via-ha

**Comment by ESP4Ever** [comment]
Hello!

Is it possible to add an icon on a notification popup somehow, like ⚠️ ☢️ 🔔 🔥 ⚡ 💧 etc (standard emoji symbols) ?
Thanks!

**Reply by felipebraz** [reply]
Also interested on it
